src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -105,7 +109,7 @@
       <Link>Common\System\HResults.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'netcoreapp1.2corert'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'netcoreapp1.2corert'">
```

Comment:
>'netcoreapp1.2corert' [](start = 109, length = 21)

I guess this should be part of the coret vertical when we add that? not sure why I only see it on this one.